README.md

```diff
@@ -70,1178 +70,68 @@ The output will be a header file, the contents hereof must be matched appropriat
 
 
 # Usage
-The [`Client`](./concordium-java-sdk/blob/main/concordium-sdk/src/main/java/com/concordium/sdk/Client.java) is the main entrypoint for the SDK.
-It facilitates an API for communicating with the [node](https://github.com/Concordium/concordium-node).
-The `Client` must be initialized with a `Connection` which holds information of the node URL, node port, password etc.
+The [`ClientV2`](./concordium-java-sdk/blob/main/concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java) is the main entrypoint for the SDK.
+It facilitates an API for communicating with the [node](https://github.com/Concordium/concordium-node) through the V2 API.
+The `ClientV2` must be initialized with a `Connection` which holds information of the node URL, node port and a timeout for the connection.
```

Comment:
I think technically the port is part of the URL

Reply:
Fixed!